submissions/merowing/js-post-oop/index.js

```diff
@@ -0,0 +1,94 @@
+import { print } from './js/lib.js';
+
+class Creature {
+   constructor(name, gender, species, hands) {
+      this.species = species;
+      this.name = name;
+      this.gender = gender;
+      this.hands = hands;
+   }
+
+   message() {
+      return `${this.species}; <strong>${this.name}</strong>; ${this.gender}; ${this.hands};`;
```

Comment:
Make this fragment DRY

Reply:
put all elements to array and then convert to string